petrel/petrel/cmdline.py

```diff
@@ -42,8 +42,9 @@ def submit(sourcejar, destjar, config, venv, name, definition, logdir, extrastor
     storm_class_path = [ subprocess.check_output(["storm","classpath"]).strip(), destjar ]
     if extrastormcp is not None:
         storm_class_path = [ extrastormcp ] + storm_class_path
-    storm_home = os.path.dirname(os.path.dirname(
-        subprocess.check_output(['which', 'storm'])))
+    storm_home = os.environ['STORM_HOME']
```

Comment:
Because in our environment the storm binary is linked, which creates a reference to /usr. STORM_HOME seems to be use elsewhere?